### DISCUSSION TITLE
Load only selected entities on wear os

### DISCUSSION THREAD

**Discussion by danielholm** [home-assistant/frontend#15336] [Other feature requests]
Hi all,

I very much enjoy the Wear OS app, but it is a bit sluggish to start, having to load all of my entities. Being able to favourite enitites is a nice function, but at times when I want to know the state I do reach for my phone instead.

The shortcuts tile is however a great addition. 

What if one could choose the entities being loaded on the Wear app in a way much like choosing entities to send to Google home, in the Cloud integration?

Some discussion done previously here https://www.reddit.com/r/homeassistant/comments/10h7z8h/home_assistant_coming_for_android_auto/j579jek?utm_medium=android_app&utm_source=share&context=3

**Comment by dshokouhi** [comment]
> What if one could choose the entities being loaded on the Wear app in a way much like choosing entities to send to Google home, in the Cloud integration?

What you are describing is exactly what the favorite entities feature allow you to do. You select what you want to load in the app and they show up first. They also show the state so not sure what the purpose of this request is. Also the correct place for this request is with the Android app and not the frontend.
